sound/soc/sof/intel/hda.c

```diff
@@ -499,6 +499,27 @@ static const struct sof_intel_dsp_desc
 	return chip_info;
 }
 
+irqreturn_t hda_dsp_interrupt_handler(int irq, void *context)
+{
+	if ((hda_dsp_ipc_irq_handler(irq, context) == IRQ_WAKE_THREAD) ||
+	    (hda_dsp_stream_interrupt(irq, context) == IRQ_WAKE_THREAD))
```

Comment:
how about change hda_dsp_stream_interrupt to hda_dsp_stream_irq_handler like hda_dsp_ipc_irq_handler

Reply:
Thanks @RanderWang . I renamed them to is_stream_irq() and is_ipc_irq()